src/stores/game.js

```diff
@@ -387,7 +396,7 @@ export const useGameStore = defineStore('game', {
               this.myPNum = res.pNum;
               this.updateGame(res.game);
               this.successfullyJoined({
-                username: res.playerUsername,
+                username: res.username, // was playerUsername but it doesn't exist 
```

Comment:
Same idea with my above comments: comments that explain why you're making a specific change should go into the PR comments rather than the code comments because they are no longer helpful once the code is merged. Like if someone a year from now reads this comment it won't matter or be helpful that it used to be `res.playerUsername` and that this was incorrect. 

But it is helpful to include in the PR comments because it's relevant to the reviewer to have that perspective. So thanks for including the context!
```suggestion
                username: res.username, // was playerUsername but it doesn't exist 
```

Reply:
yeo, ok, got it.  I didn't know about this part or the PR where we can comment every change.  It makes total sense!